app/assets/javascripts/showMore.js

```diff
@@ -0,0 +1,21 @@
+/*
+*  Inspired by https://css-tricks.com/text-fade-read-more/
+*/
+function showMore() {
```

Comment:
This is a crazy indentation! :P copying and pasting from Stack Overflow :P

Please use 2 spaces and etc.

Reply:
That isn't how it's showing to me o.O Damn tabs...

Reply:
Yeah, it happens...a lot :)